src/v2/view-builder/views/ov/ChallengeAuthenticatorDataOktaVerifyView.js

```diff
@@ -7,7 +7,42 @@ import { Body as SelectAuthenticatorVerifyViewBody } from '../SelectAuthenticato
 import { AUTHENTICATOR_KEY } from '../../../ion/RemediationConstants';
 
 const Body = SelectAuthenticatorVerifyViewBody.extend({
+  initialize() {
```

Comment:
Q: The `Send Push` view seems to be an existing view.. can we just reuse `ChallengePushVIew.js` ?  Why do we need to modify this view.. ?

Reply:
there is no polling involved for this flow ?

Reply:
`ChallengePushView` is shown when the user selects 'Get a push notification' from the auth list (usecase 1). 
`Send push` is a new view (not existing now) thats shown only when the user has previously opted for ov push authentication by  unchecking the checkbox ('Send push automatically') (usecase 2). 
They both use different remediation form and `ChallengePushView` cannot be reused.

`ChallengeAuthenticatorDataOktaVerifyView` is being modified because this and the new 'Send push' view is backed up by the same remediation form with slight difference.

Reply:
Got it.. thanks for clarifying.. If possible just create another view instead of updating `ChallengeAuthenticatorDataOktaVerifyView`